docs/tigergraph.md

```diff
@@ -0,0 +1,16 @@
+# Quick launch TigerGraph and graph-app-kit
+This guide walks through setting up a TigerGraph database to configure with `graph-app-kit`.
+
+## Setup TigerGraph
+Head over to [TG Cloud](https://tgcloud.io/) and create an account
+Follow along to the steps in this video (it is slightly outdated so the pages may look different, but the steps are the same)
+[![https://raw.githubusercontent.com/akash-kaul/static_media/main/Screen%20Shot%202021-01-18%20at%203.20.04%20PM.png](https://www.youtube.com/watch?v=JARd9ULRP_I)]
+
+### Fraud Graph
+1. Click **Create Solution** and choose the **Fraud and Money Laundering Detection** starter kit.
+2. Follow the steps shown to finish creating your solution.
+3. Wait for the graph to be created (takes ~5 minutes).
+4. Launch your graph in [GraphStudio](https://www.tigergraph.com/graphstudio/).
+5. Select your graph from the top dropdown.
+6. Head over to **Load Data** and hit the *play button* to load the data into the graph.
+7. Install all queries
```

Comment:
This doc should also detail the g-a-k steps: modify env file / fill in demo form, ...?

Reply:
and for gak, can reference (1) follow one-click launch (docs/setup.py) or manual (docs/setup-manual.py)

Reply:
The Tiger Graph Samples are using stremlit Text Box widgets for credentials , the env are for default values like 
https://<box>.i.tgcloud.io